### ISSUE TITLE
Classification loss increasing on new data set?

### ISSUE THREAD

**Issue by shreyaskamathkm** [yhenon/pytorch-retinanet#16]
Hi,

I am using [ UEC 256 ](http://foodcam.mobi/dataset256.html) database which gives bounding boxes across food items. I split the dataset into 75:25 ratio for training and validation respectively, and currently using CSV loader to load the dataset with RESNET 50 for training. My batch size is 2 and I am running it on  Nvidia Tesla P100.

While training, the loss seems to drop as expected, but while using validation the classification loss seems to increase while the regression loss is almost around the same point, I think it is overfitting the data. Any idea how to solve this?  Do I have to change the normalizing and unnormalizing factors according to my dataset? 

Thank you. Any help is appreciated.

**Comment by shreyaskamathkm** [comment]
Hi,

I am so sorry to bring this up so late. That database does not have image sets labelled as training and validation. So I had to create it manually. While creating it, I somehow missed shuffling the data. So the training dataset had classes 1:200 and validation had 201:255. I realized it when you asked me for a sample of my annotations. I have been training for almost 5 days now without realization. I will fix that asap and let you know, how it works.

Also, is normalization w.r.t the database necessary or can I keep the same normalization numbers as you provided? 

Thank you very much! Will keep you posted.

**Comment by foralliance** [comment]
@shreyaskamathkm 
@yhenon 
HI
Thank you for your analysis！！！

I've noticed that you've talked a lot about **normalization**. what exactly does that mean??
       
In `dataloader.py`
```
self.mean = np.array([[[0.485, 0.456, 0.406]]])
self.std = np.array([[[0.229, 0.224, 0.225]]])
```

In `losses.py`
`targets = targets/torch.Tensor([[0.1, 0.1, 0.2, 0.2]]).cuda()`


or something else??

many many thanks!!!
